gradle/spotbugs-exclude.xml

```diff
@@ -1,35 +1,36 @@
 <FindBugsFilter>
-  <!-- For all code !-->
-  <Match>
-    <Or>
-      <Bug pattern="THROWS_METHOD_THROWS_CLAUSE_BASIC_EXCEPTION"/>
-      <Bug pattern="THROWS_METHOD_THROWS_CLAUSE_THROWABLE"/>
-      <Bug pattern="THROWS_METHOD_THROWS_RUNTIMEEXCEPTION"/>
-      <!-- Temporarily excluded. See https://github.com/spotbugs/spotbugs/issues/1694 -->
-      <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE" />
-    </Or>
-  </Match>
+    <!-- For all code !-->
+    <Match>
+        <Or>
+            <Bug pattern="THROWS_METHOD_THROWS_CLAUSE_BASIC_EXCEPTION"/>
+            <Bug pattern="THROWS_METHOD_THROWS_CLAUSE_THROWABLE"/>
+            <Bug pattern="THROWS_METHOD_THROWS_RUNTIMEEXCEPTION"/>
+            <!-- Temporarily excluded. See https://github.com/spotbugs/spotbugs/issues/1694 -->
+            <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"/>
+            <Bug pattern="SAME_NAME_BUT_DIFFERENT"/>
```

Comment:
Only this line was added.

Reply:
Sorry, do we need to format this file?

Reply:
I updated and removed the formatting. Some XML plugins in IntelliJ were causing this. (even though format on save is disabled)